radicle-cli/src/commands/patch.rs

```diff
@@ -50,9 +52,19 @@ pub enum OptPatch {
     Patch(PatchId),
 }
```

Comment:
Maybe it's time now to get rid of this type, and just use `Option<PatchId>` (I mentioned this in the previous PR)

Reply:
This is best done with refactoring patch::create::run().  The function is used to create or update a patch, and the action depends on this value.

We should change OptPatch -> Option<PatchID>, but also split patch::create::run() into a 'create' and 'update' function.

Reply:
Hmm yeah I think that makes sense! In that case let's leave it for the next PR :stuck_out_tongue:

Reply:
It looks to deserve its own PR IMHO.